src/calculateRentalCost.js

```diff
@@ -21,6 +21,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let price;
+  if (days >= 7) {
+    price = days * 40 - 50;
```

Comment:
we can return it at once
```suggestion
    return days * 40 - 50;
```